extensions/wikia/AdEngine/js/lookup/prebid/prebid.js

```diff
@@ -105,13 +105,36 @@ define('ext.wikia.adEngine.lookup.prebid', [
 	}
 
 	function getSlotParams(slotName) {
-		var params;
+		var slotParams;
+
+		if (win.pbjs && typeof win.pbjs.getBidResponses === 'function') {
+			var params = win.pbjs.getBidResponses(slotName) || {};
+
+			if (params && params[slotName] && params[slotName].bids && params[slotName].bids.length) {
+				var bidParams,
+					priorities = adaptersRegistry.getPriorities();
+
+				params[slotName].bids.forEach(function (param) {
```

Comment:
This piece of logic is critical for our business. Can we add some unit tests to make sure everything works as expected?
Additionally, it's really hard to test it in the real environment.

Reply:
Good suggestion. Added.